harness/determined/_trial_controller.py

```diff
@@ -64,6 +64,7 @@ def pre_execute_hook(env: det.EnvContext, hvd_config: horovod.HorovodContext) ->
     @abc.abstractmethod
     def from_trial(
         trial_inst: "det.Trial",
+        prof: profiler.ProfilerAgent,
```

Comment:
So I'll leave it up to you since it doesn't really matter, but I would prefer we call the variable `profiler`

Reply:
I just don't like shadowing package names

Reply:
That's fair